### PR TITLE
Implemented unit tests to verify the correct working of sortPoints.js

### PR DESCRIPTION
**Pull request by ShimElliott**
Includes a test that verifies that a single point input produces a single point output.
Includes a test that verifies that an unsorted list is sorted according to the x value, in ascending order.

### CODE REVIEW

**Comment by ShimElliott** [comment]
Also added jest and the "test": "jest" command to scripts in the package.json.

File: src/lib/sortPoints.test.js [hunk 1]
```diff
@@ -0,0 +1,48 @@
+const sortPoints = require("./sortPoints")
+
+test ('an input of one point returns same point', function() {
```
Comment (Clarhunt) [inline src/lib/sortPoints.test.js:3]: Your two tests seem to do a good job at testing inputs for the sortPoints() function. So far I think your two tests seem very valid and are a great start. It would also be a good idea to test points with different y values and constant x values and test points with different points with all different values for x and y. So far so good :)

**Review by dualsm — COMMENTED** [review]
Looks good to me!

**Review by Clarhunt — APPROVED** [review]
Looks good so far. Probably needs a few more tests to make sure you cover all inputs.

**Review by dualsm — APPROVED** [review]
approved, starting commit seems simple & satisfies sortPoints function. Are there edge cases like negative numbers that would make this fail? Maybe we can add those later, I don't see them causing problems but we might want to verify the behavior.
